start-workflow-from-archive/StartWorkflow.py

```diff
@@ -0,0 +1,106 @@
+import argparse
```

Comment:
You might want to add a shebang

Reply:
You might want to take a look at and adhere to [PEP 8 (The official Python style guide)](https://www.python.org/dev/peps/pep-0008/). This includes naming [your python files using lowercase characters](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) and doing some inline changes like sticking to line length < 80 characters. Try using `flake8` to swiftly check your files.

Reply:
done